### DISCUSSION TITLE
Add a functionality similar to Apple Photos "Memories" (automatically curated collections displayed as videos with music)

### DISCUSSION THREAD

**Discussion by mapperfr** [ente-io/ente#1721] [Enhancements]
Apple Photos provide a functionality called Memories, described here: https://support.apple.com/en-us/118279

> The Photos app recognizes significant people, places, and events in your library, then presents them in curated collections called Memories. You can also create your own Memories, and share them with friends and family.

A similar functionality in ente would be amazing.

**Comment by NickDiMucci** [comment]
You probably won't be able to license Hans Zimmer or The Beatles for the background tracks (or maybe you will? who knows!) but even with open license tracks, this would be great 💚
